tests/integration/supplychain/policy.yaml

```diff
@@ -15,11 +15,13 @@
 apiVersion: audit.k8s.io/v1 # This is required.
 kind: Policy
 # Don't generate audit events for all requests in RequestReceived stage.
```

Comment:
```suggestion

```
That comment seems to be irrelevant now as it was on the `omitStages` portion that was removed.